lib/pretty_text/helpers.rb

```diff
@@ -116,6 +110,29 @@ def category_tag_hashtag_lookup(text)
       end
     end
 
+    def hashtag_lookup(slug, cooking_user, types_in_priority_order)
+      # This is _somewhat_ expected since we need to be able to cook posts
+      # etc. without a user sometimes, but it is still an edge case.
+      if cooking_user.blank?
```

Comment:
do we need to check for blank here?

Reply:
Yes, this is what the comment is saying. Sometimes it is blank because we can cook a post or message without a user (for example a post goes through multiple cooks to validate things before it does the final cook). It is a little weird, but it is safer than the old system which had no guardian checks at all for these categories/hashtag lookups

Reply:
I mean do we need blank or can we just check for nil?